src/Controller/LogOperationsController.php

```diff
@@ -93,7 +93,6 @@ private function getStatusMessage($status, $logRow)
                 return "";
             case Status::TRANSFER_FAILED:
             case Status::WITHDRAW_FAILED:
-            case Status::WITHDRAW_FAILED:
             case Status::WITHDRAW_CANCELED:
                 return '<button type="button" class="btn btn-info btn-xs vendor-notice" data-container="body" data-toggle="popover" data-placement="bottom" data-content="' .
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 173 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS103)